package.json

```diff
@@ -21,7 +21,9 @@
   ],
   "dependencies": {
     "angular": "1.x",
-    "angular-sanitize": "1.x"
+    "angular-sanitize": "1.x",
+    "grunt": "0.4.5",
+    "grunt-cli": "1.2.0"
```

Comment:
I don't, that was a mistake.






On Sun, Apr 17, 2016 at 10:10 PM -0700, "Ritesh Kumar" <notifications@github.com> wrote:





> @@ -21,7 +21,9 @@
>    ],
>    "dependencies": {
>      "angular": "1.x",
> -    "angular-sanitize": "1.x"
> +    "angular-sanitize": "1.x",
> +    "grunt": "0.4.5",
> +    "grunt-cli": "1.2.0"

Why do you need grunt and grunt-cli here ? they should be in `devDependencies`

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/ritz078/ng-embed/pull/36/files/579aebec13fc46146b46a45b0caee6124b2e5861#r60006463